test/README.md

```diff
@@ -180,6 +181,26 @@ Note: gdb attach step may require `sudo`
 Util tests can be run locally by running `test/util/bitcoin-util-test.py`.
 Use the `-v` option for verbose output.
 
+### Lint tests
+
+#### Dependencies
+
+The lint tests require codespell and flake8. To install: `pip3 install codespell flake8`.
```

Comment:
Just noting that I think it should not specify versions, this way new lint errors.can be detected and the source fixed accordingly.